cmake/RunTests.cmake

```diff
@@ -16,7 +16,13 @@ endif()
 
 if(DEFINED ENV{TEST_FILE})
   set(TEST_PATH "$ENV{TEST_FILE}")
-  set(rel_test_path "${TEST_PATH}")
+  if(WIN32)
+    # Remove the drive letter colon(:) because Windows must not contain the
+    # colon(:) in the file name.
+    string(REPLACE ":" "" rel_test_path "${TEST_PATH}")
```

Comment:
would this change `C:/foo` to `C/foo` ? And that still works?

Reply:
Ah.. it might be passed in as a absoute file (and is then not made relative).
Maybe it should be made relative always?

Reply:
> Ah.. it might be passed in as a absoute file (and is then not made relative).

Yes I set the absolute path to `TEST_FILE`.

> Maybe it should be made relative always?

I changed it to convert to relative path. Should I make any other changes besides Windows?

>  Should we remove all special chars?

I do not think that it is necessary to set a file name that can not normally be created to `TEST_FILE`, as that does not make sense. However, if it corresponds to all legal file names, conversion of the network path(Begins with `\\\\`) may be necessary.

I thought that relative path can not be used because `TEST_FILE` was set relative to `${BUILD_DIR}`. I realized that I needed to specify a relative path from `${WORKING_DIR}`.